packages/@react-spectrum/table/test/TestTableUtils.test.tsx

```diff
@@ -31,7 +31,7 @@ let columns = [
 describe('Table ', function () {
   let onSelectionChange = jest.fn();
   let onSortChange = jest.fn();
-  let testUtilRealTimer = new User();
+  let testUtilRealTimer = new User({advanceTimer: async (waitTime) => await new Promise((resolve) => setTimeout(resolve, waitTime))});
```

Comment:
why async?

Reply:
yeah it doesn't need the async await here, copy pasta from Analytic's real timer wait util just to mimic their setup as closely as possible to make sure the test-utils would work with their setup. The `act` happens by whatever calls `advanceTimer` aka https://github.com/adobe/react-spectrum/blob/f75223a04382fd410636d615cfb859fd2452098e/packages/%40react-aria/test-utils/src/events.ts#L43